firebase-admin/runtime/pom.xml

```diff
@@ -38,6 +39,19 @@ quarkus-google-cloud-common-grpc and Quarkus includes the netty library -->
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-grpc-common</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-vertx-http</artifactId>
+        </dependency>
```

Comment:
Is it realy needed ? Isn't it a transitive dependency of quarkus-security ?

Reply:
Doesn't seem like it is.